lib/mode-signin/index.js

```diff
@@ -212,6 +215,33 @@ SigninPanel.prototype.bindAll = function() {
 };
 
 /**
+ * Enables/disables the recaptcha widget
+ *
+ * @param {Boolean} required
+ * @return {RecaptchaWidget}
+ * @public
+ */
+
+SigninPanel.prototype.requireRecaptcha = function(required) {
+  if (required) {
+    var self = this;
+
+    var recaptcha_container = this.query('.a0-recaptcha');
+    var submit_button =  this.query('button[type=submit]');
+    var recaptcha_token_holder = this.query('input[name=recaptcha-token]');
+
+    submit_button.attr('disabled', true);
+    
+    this.recaptcha.show(recaptcha_container[0], function(token) {
```

Comment:
good question. I thought the same thing, but the docs didn't talk about that situation. I'll research a little about it. 
Anyway, I think it would be a really uncommon situation and a critic issue to Google. I mean, the recaptcha's script gets loaded but the render methods fail.